.github/workflows/test.yaml

```
@@ -261,6 +261,55 @@ jobs:
         fail_ci_if_error: true
 
 
+  fips-tests:
+    # Defined the host in which the container is executed
+    runs-on: ubuntu-latest
+    # Define container inside which the steps are executed
+    container:
+      image: photon:5.0
+      env:
+        NODE_ENV: development
+      options: --cpus 2
```

Would something like this trigger the container to be started as non root.

```suggestion
      options: --cpus 2 --user=runner
```

I am not very familiar with Docker or GHA containers... I was expecting the container to be started as non-root by defaul.

---

Same here. Here I want to install few packages in the container in the beginning, this should be done as root.
Later tests should be run as regular user. So, I'm not sure how to achieve it in GHA. I tried few things, didn't work.